x-pack/elastic-agent/spec/endpoint.yml

```diff
@@ -57,4 +57,4 @@ rules:
     - output
     - revision
 
-when: HasAny('fleet') && HasItems(%{[inputs]}) && HasNamespace('output', 'elasticsearch')
+when: length(${fleet}) > 0 and length(${inputs}) > 0 and hasKey(${output}, 'elasticsearch')
```

Comment:
I really like that our spec files now also use EQL conditions. Makes it much easier to just have 1 implementation.